pricegraph/src/orderbook.rs

```diff
@@ -222,6 +328,18 @@ mod tests {
         }};
     }
 
+    impl Orderbook {
```

Comment:
Should this only be available in tests?

Reply:
Yes, its a testing utility method.

Reply:
Oh sorry, I missed it was under the test module.